.github/workflows/node.yml

```diff
@@ -147,6 +162,17 @@ jobs:
     #           run: npm test -- -t "set and get flow works"
     #           working-directory: ./node
 
+    #         - name: Upload test reports
+    #           if: always()
+    #           continue-on-error: true
+    #           uses: actions/upload-artifact@v4
+    #           with:
+    #               name: test-report-node-${{ matrix.engine.type }}-${{ matrix.engine.version }}-macos
+    #               path: |
+    #                   node/test-report*.html
+    #                   utils/clusters/**
+    #                   benchmarks/results/**
```

Comment:
Since you have commented out above lines, do we need to keep these?

Reply:
Yes, it is a part of a CI job which is completely commented out in #1881. I hope it will be uncommented soon.

Reply:
It will